packages/scandipwa/src/component/MyAccountCreateAccount/MyAccountCreateAccount.component.js

```diff
@@ -24,12 +25,7 @@ export class MyAccountCreateAccount extends PureComponent {
         onCreateAccountSuccess: PropTypes.func.isRequired,
         handleSignIn: PropTypes.func.isRequired,
         showTaxVatNumber: PropTypes.string.isRequired,
-        vatNumberValidation: PropTypes.array.isRequired,
-        defaultValues: PropTypes.object
-    };
-
-    static defaultProps = {
-        defaultValues: {}
+        vatNumberValidation: PropTypes.array.isRequired
     };
```

Comment:
Please, do not remove existing functinality for no reason. Ideally I would make the container update these fields if state is present, so no logic is left in component.

Reply:
The reason for removal of default props is that the component doesn't use `defaultValues` anymore. This line has also been removed:
```const { defaultValues: { email = '' } } = this.props;```

And props `defaultValues` is not passed to `MyAccountCreateAccount` by any other component any more.